netty/src/main/java/io/grpc/netty/GrpcSslContexts.java

```diff
@@ -133,6 +134,7 @@ public static SslContextBuilder forServer(
    * Set ciphers and APN appropriate for gRPC. Precisely what is set is permitted to change, so if
    * an application requires particular settings it should override the options set here.
    */
+  @CanIgnoreReturnValue
```

Comment:
Because it is included in the package, it applies to all classes.  See package-info.java